hack/templates/excluder-template

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
I have a feeling @smarterclayton would prefer this live somewhere in contrib rather than hack.